mongo/integration/mtest/mongotest.go

```diff
@@ -666,13 +666,30 @@ func verifyTopologyConstraints(topologies []TopologyKind) error {
 	return fmt.Errorf("topology kind %q does not match any of the required kinds %q", testContext.topoKind, topologies)
 }
 
+func verifyServerParametersConstraints(serverParameters map[string]bool) error {
+	if serverParameters == nil {
+		return nil
+	}
+
+	for param, expectedValue := range serverParameters {
+		actualValue, ok := testContext.serverParameters.Lookup(param).BooleanOK()
+		if !ok || expectedValue != actualValue {
+			return fmt.Errorf("server parameter %v does not match required value %v", param, actualValue)
```

Comment:
nit: for readability, use `%q` for strings like `param`. Go will add quotes around them in the final message so it's easy to see what the parameter name is.

Reply:
I should read up on `fmt` a little more 🕵️‍♂️ . Changed param to `%q` here and added `%q` in a few other places.